packages/contract/src/DaimoVerifier.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.12;
+
+import "openzeppelin-contracts/contracts/proxy/utils/UUPSUpgradeable.sol";
+import "openzeppelin-contracts-upgradeable/contracts/access/OwnableUpgradeable.sol";
+import "openzeppelin-contracts/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+
+import "p256-verifier/WebAuthn.sol";
+
+/// Proxy using the UUPSUpgradeable pattern. Named for Etherscan verification.
+contract DaimoVerifierProxy is ERC1967Proxy {
+    constructor(
+        address _logic,
+        bytes memory _data
+    ) ERC1967Proxy(_logic, _data) {}
+}
+
+struct Signature {
+    bytes authenticatorData;
+    string clientDataJSON;
+    uint256 challengeLocation;
+    uint256 responseTypeLocation;
+    uint256 r;
+    uint256 s;
+}
+
+// Proxies a signature verification call to the Webauthn library.
+// This contract exists only to enable upgradability of the Webauthn
+// library that is in control of the owner rather than account.
```

Comment:
what do you mean by "that is in control of ..."?

Maybe just "to enable temporary upgradeability of the Webauthn library. After further audit(s), ownership will be burned, making this verifier library immutable."